src/openms/source/SYSTEM/UpdateCheck.cpp

```diff
@@ -73,26 +73,33 @@ namespace OpenMS
     platform = "Win";
 #elif __APPLE__
     platform = "Mac";
-#else
+#elif __unix__
```

Comment:
not sure about that part.
XDG mentions KDE, GNOME, XServer and alike, and this sounds like Linux to me (`__linux__`).
However, `__unix__` seems to be defined on our debian as well, which I find very suprising (linux != unix, right? :-) )

Reply:
I've added a check for __linux__ first and then __unix__ if its not linux. According to http://web.archive.org/web/20191012035921/http://nadeausoftware.com/articles/2012/01/c_c_tip_how_use_compiler_predefined_macros_detect_operating_system all linux variants have __unix__ set.